shared/test/textChatView_test.js

```diff
@@ -203,26 +203,6 @@ describe("loop.shared.views.TextChatView", function() {
           .to.eql(2);
     });
 
-    it("should show timestamps if they are 1 minute apart (SENT)", function() {
```

Comment:
I have a feeling we'll have SENT messages back soon, so probably best to it.skip and comment for now instead of just deleting.